aiida/schedulers/plugins/lsf.py

```diff
@@ -151,6 +151,15 @@ def accepts_default_mpiprocs_per_machine(cls):
         """
         return False
 
+    @classmethod
+    def validate_resources(cls, **kwargs):
+        """Validate the resources against the job resource class of this scheduler.
+
+        :param kwargs: dictionary of values to define the job resources
+        :return: attribute dictionary with the parsed parameters populated
+        :raises ValueError: if the resources are invalid or incomplete
+        """
+
```

Comment:
Guessing `@abstractclassmethod` didn't actually work, since this was not picked up before? @sphuber I guess this should probably have some content

Reply:
Yeah that is weird indeed. I guess the validation is already done in the constructor. We can just move that logic to the `validate_resources`. To be safe as to not break anything, you can always still call `self.validate_resources(**kwargs)` from the constructor, which is what the `NodeNumberJobResource` also does.

Reply:
done